neonvm/config/controller/deployment.yaml

```diff
@@ -59,6 +59,8 @@ spec:
         # * cache.direct=on    - use O_DIRECT (don't abuse host's page cache!)
         # * cache.no-flush=on  - ignores disk flush operations (not needed; our disks are ephemeral)
         - "--qemu-disk-cache-settings=cache.writeback=on,cache.direct=on,cache.no-flush=on"
+        - "--failure-pending-period=1m"
+        - "--failing-refresh-interval=1m"
```

Comment:
Could we have this slightly smaller? e.g. 15 seconds? asking only so that the logs can have higher granularity than metrics (and so that the metrics are delayed by at most 15s, rather than at most 1 minute)

Reply:
Implemented in 5454f0f94f9c9b56e2ab0c971ed8701e46c9ddd4